core_data_modules/cleaners/codes/somalia_codes.py

```diff
@@ -45,7 +45,7 @@ class SomaliaCodes(object):
     GARBAHAAREY = "garbahaarey"
```

Comment:
With changes like this - could you also include what the source of the changes is? I obviously have no way of reviewing spelling, but it would be nice to understand a bit more about why we're updating them.